automated-tests/src/main/java/com/hp/ov/sdk/network/EthernetNetworkResource.java

```diff
@@ -112,6 +117,30 @@ public int getAssociatedUplinkGroups(String resourceID) {
         return -1;
     }
 
+    public String patchEthernetNetwork(String id) {
+        Network network = new Network();
+        network.setName(resourceProperties.get("name"));
+
+        Patch patch = new Patch();
+
+        patch.setOp(Patch.PatchOperation.valueOf(resourceProperties.get("op")));
+        patch.setPath(resourceProperties.get("path"));
+        List<String> scopeUris = network.getScopeUris();
+        scopeUris.add(SCOPES_URI + "/" + getScopeId(resourceProperties.get("value")));
+        patch.setValue(scopeUris);
+
+        return objToString(client.patch(id, patch));
+
+    }
+
+    private String getScopeId(String scopeName) {
+        ScopeClient scopeClient = oneViewClient.scope();
+
+        Scope scope = scopeClient.getByName(scopeName).get(0);

```

Comment:
Please add a verification if scope exists, to avoid a NPE.

Reply:
Done.